docs/bis/ec/kindofquantity.md

```diff
@@ -59,8 +59,11 @@ The below represents the string literal syntax. All italicized values are to be
   - `formatName(precision)[First Unit][Second Unit][Third Unit][Fourth Unit]`
 - Overrides may not change units if any are originally defined in the base format, they may only change the display label.
 - Up to 4 per format string
+- As unit labels are optional, they can be kept unset or they can be set to an empty string.
+- When a unit label is not specified in the override, the label will be defaulted to the display label of the unit. See example [Null unit label in override](#format-overrides-with-nullempty-unit-labels).
```

Comment:
I believe it will fall back to the label specified by the underlying format if it exists ... test to confirm this first